README.old.md

```diff
@@ -0,0 +1,33 @@
+# july-challenge-tbd
```

Comment:
Oh I'm seeing now that the old README content is here instead, oops! Is there a reason to have two separate ones?

Reply:
I was so confused for a sec as I was reading through your comments LOL  I thought I pushed the wrong code for a sec. The different READMEs are from my old one and the new one from create-react-app. I can combine them, thanks for the notice!